apis/elasticloadbalancing/v1alpha1/elb_attachment_types.go

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright 2020 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	runtimev1alpha1 "github.com/crossplane/crossplane-runtime/apis/core/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ELBAttachmentParameters define the desired state of an AWS ELBAttachment.
+type ELBAttachmentParameters struct {
+	// Name of the Elastic Load Balancer to which the instances will attach.
+	// +optional
+	ELBName string `json:"elbName,omitempty"`
```

Comment:
Can we add a reference for this resource?

Reply:
Thanks for the catch, added the reference.